src/test/suite/notebook.test.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+/* eslint-disable @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires */
+import { assert } from 'chai';
+import { window, commands, CompletionList, Position, Disposable } from 'vscode';
+import {
+    canRunNotebookTests,
+    closeNotebooksAndCleanUpAfterTests,
+    insertCodeCell,
+    createEmptyPythonNotebook,
+    traceInfo,
+    initializeTestWorkspace
+} from './helper';
+
+/* eslint-disable @typescript-eslint/no-explicit-any, no-invalid-this */
+suite('Notebook tests', function () {
+    const disposables: Disposable[] = [];
+    this.timeout(240_000);
+    suiteSetup(async function () {
+        this.timeout(120_000);
+        if (!canRunNotebookTests()) {
+            return this.skip();
+        }
+        await initializeTestWorkspace();
+    });
+    // Use same notebook without starting kernel in every single test (use one for whole suite).
+    setup(async function () {
+        traceInfo(`Start Test ${this.currentTest?.title}`);
+        await createEmptyPythonNotebook(disposables);
+        process.env.VSC_JUPYTER_IntellisenseTimeout = '30000';
+        traceInfo(`Start Test (completed) ${this.currentTest?.title}`);
+    });
+    teardown(async function () {
+        traceInfo(`Ended Test ${this.currentTest?.title}`);
+        delete process.env.VSC_JUPYTER_IntellisenseTimeout;
+        await closeNotebooksAndCleanUpAfterTests(disposables);
+        traceInfo(`Ended Test (completed) ${this.currentTest?.title}`);
+    });
+    suiteTeardown(() => closeNotebooksAndCleanUpAfterTests(disposables));
+    test('Add some cells and get completions', async () => {
```

Comment:
First simple test - add some cells and get completions. 

I'll be adding more tests here after this submission to verify things like:
- Splitting cells
- Deleting cells
- Having markdown cells
- Typing into the editor (I hope I can do this somehow)